launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -33,9 +34,20 @@
     <arg name="image_number" value="$(var image_number)"/>
   </include> -->
 
+  <!-- Pointcloud map filter -->
+  <include file="$(find-pkg-share compare_map_segmentation)/launch/voxel_based_compare_map_filter.launch.xml" if="$(var use_pointcloud_map)">
+    <arg name="input" value="$(var input_pointcloud)" />
+    <arg name="input_map" value="/map/pointcloud_map" />
+    <arg name="output" value="pointcloud_map_filtered/pointcloud" />
+    <arg name="distance_threshold" value="0.5" />
+  </include>
```

Comment:
I think it needs group tag here otherwise arg "output" in detection_by_tracker is changed to "pointcloud_map_filtered/pointcloud".
```suggestion
  <group>
    <include file="$(find-pkg-share compare_map_segmentation)/launch/voxel_based_compare_map_filter.launch.xml" if="$(var use_pointcloud_map)">
      <arg name="input" value="$(var input_pointcloud)" />
      <arg name="input_map" value="/map/pointcloud_map" />
      <arg name="output" value="pointcloud_map_filtered/pointcloud" />
      <arg name="distance_threshold" value="0.5" />
    </include>
  </group>
```